lightllm/server/io_struct.py

```diff
@@ -52,6 +52,12 @@ def __init__(self, batch_id, reqs: List[Req]):
         self.batch_id = batch_id
         self.reqs = reqs
         self.id_to_reqs = {req.request_id: req for req in reqs}
+        self.batch_input_tokens = 0
+        for req in self.reqs:
+            self.batch_input_tokens += req.input_len
+
+    def input_tokens(self):
+        return self.batch_input_tokens
```

Comment:
After filtering or merging, the consistency of `batch_input_tokens` will be broken.
I suggest just re-computing it every time `input_tokens` get called

Reply:
> After filtering or merging, the consistency of `batch_input_tokens` will be broken. I suggest just re-computing it every time `input_tokens` get called

done